e2e/_suites/kubernetes-autodiscover/features/elastic_agent_standalone.feature

```diff
@@ -4,30 +4,35 @@ Feature: elastic-agent standalone
   Use Kubernetes autodiscover features in elastic-agent standalone to collect logs and metrics
 
 Scenario: Logs collection from running pod
-  Given "elastic-agent" is running with "logs generic"
+  Given "elasticsearch" is running
+    And "elastic-agent" is running with "logs generic"
```

Comment:
We could have a single `Given` clause with:

```gherkin
Given "elastic-agent" is running with "logs generic"
```

and define a `Background` clause at the beginning of the feature file:

```gherkin
Background: Setting up elasticsearch
  Given "elasticsearch" is running
```
which will be run before every scenario. Therefore you can simplify each scenario removing the initial "Given elasticsearch" step